### ISSUE TITLE
`error getting msbuild info` found via `#58`

### ISSUE THREAD

**Issue by matthid** [enricosada/dotnet-proj-info#59]
As I'm already running some custom builds including #58 I can now properly report the issues:

I just had the situation where a project was shown as "load failed" with (I formatted the message slightly to be more readable):

```
error getting msbuild info: 
    (Error MSBuildSkippedTarget, 
     Error MSBuildSkippedTarget, 
     Ok (Properties 
            [("TargetPath", ""); 
             ("IsCrossTargetingBuild", "true"); ("TargetFrameworks", "net462;netstandard2.0"); ("OutputType", "Library"); 
             ("IsTestProject", ""); ("TargetPath", ""); ("Configuration", "Debug"); ("IsPackable", "true"); ("TargetFramework", ""); 
             ("TargetFrameworkIdentifier", ""); ("TargetFrameworkVersion", ""); 
             ("MSBuildAllProjects", ";C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props;c:\proj\FAKE\src\Directory.Build.props;c:\proj\FAKE\src\app\Fake.DotNet.ILMerge\obj\Fake.DotNet.ILMerge.fsproj.nuget.g.props;c:\proj\FAKE\src\app\Fake.DotNet.ILMerge\obj\Fake.DotNet.ILMerge.fsproj.paket.props;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.props;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.props;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.SupportedTargetFrameworks.props;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FSharp.props;C:\Program Files\dotnet\sdk\2.1.506\FSharp\Microsoft.FSharp.NetSdk.props;c:\proj\FAKE\.paket\Paket.Restore.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.DefaultAssemblyInfo.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.DefaultOutputPaths.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FSharpTargetsShim.targets;C:\Program Files\dotnet\sdk\2.1.506\NuGet.targets;C:\Users\matth\AppData\Local\Temp\tmpA197.tmp.proj-info.hook.targets;C:\Program Files\dotnet\sdk\2.1.506\15.0\Microsoft.Common.CrossTargeting.targets\ImportAfter\Microsoft.TestPlatform.CrossTargeting.targets;c:\proj\FAKE\src\app\Fake.DotNet.ILMerge\obj\Fake.DotNet.ILMerge.fsproj.nuget.g.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.CrossTargeting.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.Common.targets;C:\Program Files\dotnet\sdk\2.1.506\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets"); 
             ("ProjectAssetsFile", "C:\proj\FAKE\src\app\Fake.DotNet.ILMerge\obj\project.assets.json"); ("RestoreSuccess", "True"); 
             ("Configurations", "Debug;Release"); ("TargetFrameworks", "net462;netstandard2.0"); ("RunArguments", ""); 
             ("RunCommand", ""); ("IsPublishable", "")]),
     Ok (Items
            [{Name = "Compile"; Identity = "AssemblyInfo.fs"; 
              Metadata = [
                (FullPath, "c:\proj\FAKE\src\app\Fake.Core.Context\AssemblyInfo.fs"); (Custom "Link", "")];}; 
             {Name = "Compile"; Identity = "Context.fs"; 
              Metadata = [(FullPath, "c:\proj\FAKE\src\app\Fake.Core.Context\Context.fs"); (Custom "Link", "")];}]))
```

And indeed, this particular case seems to be missing in the code:

https://github.com/enricosada/dotnet-proj-info/blob/e10b0db07b7af1c5621d934f0c31c55432909e54/src/Dotnet.ProjInfo.Workspace/ProjectCrackerDotnetSdk.fs#L153-L166

The project in question was: 
https://github.com/fsharp/FAKE/blob/9f47e9c0078507c4851371544c642d21888cc413/src/app/Fake.DotNet.Testing.VSTest/Fake.DotNet.Testing.VSTest.fsproj#L1-L26

/cc @enricosada

**Comment by matthid** [comment]
Workaround: Editing the project file will usually make it load the second time.

**Comment by matthid** [comment]
ping @enricosada :)

**Comment by enricosada** [comment]
diditn forget, trying to merge https://github.com/fsharp/FsAutoComplete/pull/424 first.
i messed up that, rewriting the PR tonight
sorry for delay

**Comment by matthid** [comment]
No worries, I just know that I easily forget about some PRs sometimes ;)

**Comment by enricosada** [comment]
should be fixed by #61 thanks a lot for awesome repro and bug report!
and obv to having improved a lot the errors loggin
